timeline/templatetags/timeline_tags.py

```diff
@@ -70,24 +69,35 @@ def get_plugin_lookup():
 def get_app_icon_html(event, plugin_lookup):
     """Return icon link HTML for app by plugin lookup"""
     url = None
-    url_kwargs = {}
-    if event.project:
-        url_kwargs['project'] = event.project.sodar_uuid
     title = event.app
     icon = ICON_UNKNOWN_APP  # Default in case the plugin is not found
 
     if event.app == 'projectroles':
         if event.project:
```

Comment:
This condition can be omitted, as the projectroles entry point URL is fixed and always has the project UUID as kwarg. Also, projectroles will not create events without an associated project.

Reply:
Implemented this as well